src/components/formio/radio.tsx

```diff
@@ -51,9 +55,16 @@ export const Radio: React.FC<RadioProps> = ({
   options,
   label,
   required = false,
+  isClearable = true,
```

Comment:
let's default this to `false`. Idiomatic react is passing booleans as `<Radio isClearable />` which enables it, and not specifying it disables it.

Furthermore, I don't think default clearable makes sense, otherwise the browsers would've built in such behaviour natively for radio buttons :)